docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,791 @@
+
+// Toolkit.Mvvm
+Messenger.Default.Unregister<MyMessage>(this);
+```
+
+#### Unregister<TMessage> ( object, object )
+
+The functionality of `Unregister<TMessage>(object, object)` can be achieved with the MVVM Toolkit's `Unregister<TMessage, TToken>(object, TToken)` method. 
+
+```csharp
+// MvvmLight
+Messenger.Default.Unregister<MyMessage>(this, nameof(MyViewModel));
+
+// Toolkit.Mvvm
+Messenger.Default.Unregister<MyMessage, string>(this, nameof(MyViewModel));
+```
+
+#### Unregister<TMessage> ( object, object, Action<TMessage> )
+
+There is no direct replacement for the `Unregister<TMessage>(object, object, Action<TMessage>)` method in the MVVM Toolkit. 
+
+Alternatively, we recommend achieving a similar functionality with the MVVM Toolkit's `Unregister<TMessage, TToken>(object, TToken)` method. 
+
+```csharp
+// MvvmLight
+Messenger.Default.Unregister<MyMessage>(this, nameof(MyViewModel), this.OnMyMessageReceived);
+
+// Toolkit.Mvvm
+Messenger.Default.Unregister<MyMessage, string>(this, nameof(MyViewModel));
+```
+
+#### Cleanup ()
+
+There is no direct replacement for the `Cleanup` method in the MVVM Toolkit. In the context of MvvmLight, `Cleanup` is used to remove registrations which are no longer alive as the implementation takes advantage of weak references. 
+
+Any calls to the `Cleanup` method can be removed.
+
+```csharp
+// MvvmLight
+Messenger.Default.Cleanup();
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+#### RequestCleanup ()
+
+There is no direct replacement for the `RequestCleanup` method in the MVVM Toolkit. In the context of MvvmLight, `RequestCleanup` is used to initiate a request to remove registrations which are no longer alive as the implementation takes advantage of weak references. 
+
+Any calls to the `RequestCleanup` method can be removed.
+
+```csharp
+// MvvmLight
+Messenger.Default.RequestCleanup();
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+#### ResetAll ()
+
+There is no direct replacement for the `ResetAll` method in the MVVM Toolkit. In the context of MvvmLight, this method is used to reset the default `IMessenger` instance by calling the static `Reset` method.
+
+Any calls to the `ResetAll()` method can be removed.
+
+```csharp
+// MvvmLight
+Messenger.Default.ResetAll();
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+### Messenger Static Methods
+
+#### OverrideDefault ( IMessenger )
+
+There is no direct replacement for the `OverrideDefault(IMessenger)` method in the MVVM Toolkit.
+
+Any calls to the `Messenger.OverrideDefault(IMessenger)` method can be removed.
+
+```csharp
+// MvvmLight
+Messenger.OverrideDefault(new Messenger());
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+#### Reset ()
+
+There is no direct replacement for the `Reset` method in the MVVM Toolkit. In the context of MvvmLight, this static method is used to reset the default `IMessenger` instance.
+
+Any calls to the `Messenger.Reset()` method can be removed.
```

Comment:
Follow up from the previous comment, we could say that in this case users would simply use the `Reset` method on the `Messenger.Default` instance directly - the result is the same in both cases anyway.

Reply:
Sorted this with previous change 👍